src/main/kotlin/de/klg71/keycloakmigration/changeControl/actions/client/UpdateClientAction.kt

```diff
@@ -51,7 +52,7 @@ class UpdateClientAction(
             protocol ?: oldClient.protocol,
             attributes ?: oldClient.attributes,
             oldClient.authenticationFlowBindingOverrides,
-            oldClient.fullScopeAllowed,
+            fullScopeAllowed ?:oldClient.fullScopeAllowed,
```

Comment:
Would you add this do the documentation in docsbuild/content/migrations/client.md #updateClient ?

Reply:
done